portal-api/src/v1/users/validation/rules/select-at-least-one-bank.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Validates that if the password is present it is at least 8 characters long
+ * @param {Object} user the existing user
+ * @param {Object} change the changes to make
+ * @returns {Array} either an empty array or an array containing an error object
+ */
+const selectAtLeastOneBank = (user, change) => {
+  if (!change.bank) {
+    return [
+      {
+        bank: {
+          order: '4',
+          text: 'Bank is required',
+        },
+      },
+    ];
+  }
+
+  if (change.bank === 'all' && (!change.roles || !change.roles.includes('admin'))) {
```

Comment:
* `all` to be picked up from constant.
* Statement can be simplified as following
```javascript
const canHaveAllBank = change.bank === CONSTANT.ALL && change.roles?.includes(CONSTANT.ADMIN);
if (canHaveAllBank) { ...
```

Reply:
Updated according to the comment.